lib/lz4hc.h

```diff
@@ -314,6 +326,12 @@ void LZ4_favorDecompressionSpeed(LZ4_streamHC_t* LZ4_streamHCPtr, int favor);
  *  - the stream was in an indeterminate state and was used in a compression
  *    call that fully reset the state (LZ4_compress_HC_extStateHC()) and that
  *    returned success
+ *
+ *  Note:
```

Comment:
This is kind of misleading. Passing a context into any `_resetStream` function should always clear the history.

Reply:
Updated.

Reply:
I think this comment overall should be cleaned up a little. Like with the LZ4 fast changes, we no longer need to distinguish between successful returns and errors: the context is in a valid state either way. So the wording above can be slightly simplified.

I also wouldn't call out `LZ4_compress_HC_continue()` specifically below, since the statement applies to all APIs. I might word the note as:

>  A stream that was last used in a compression call that returned an error may be passed to this function. However, it will be fully reset, which will clear any existing history and settings from the context.